back/src/Guide/Database/Convert.hs

```diff
@@ -0,0 +1,173 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+
+module Guide.Database.Convert
+       (
+       -- Encoders
+         boolParam
+       , boolParamNullable
+       , textParam
+       , textParamNullable
+       , traitTypeParam
+       , uidParam
+       -- Decoders
```

Comment:
I think it's better to group functions by type (bool/text/etc), not by decoder/encoder, because then it will be easier to notice if an encoder and a decoder don't match.